lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
+
 
 const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
+      let tabs=props.tab===props.selected
```

Comment:
`selectedTab` is how you sent it, change selected to selectedTab and your styles should be applied